src/widgets.ts

```diff
@@ -83,6 +85,33 @@ export abstract class Widget<
     return dispatchWidgetPostMessageEvent
   }
 
+  /**
+   * Public method to communicate with iframe widget about navigation events
+   * Can be called when the host has a 'back' event happen.
+   * This will send a post message event to the iframe widget, which if it's
+   * listening for the 'mx/navigation' event, it can make its changes and then
+   * send its own post message event back to the SDK for the 'onNavigation' callback
+   */
+  navigateBack() {
+    const iframeElement = this.iframe.contentWindow
```

Comment:
This method doesn't do anything if `iframeElement` is falsy, so let's just bail early if that's the case.

Reply:
Updated